### DISCUSSION TITLE
Not finding the pattern while calibration

### DISCUSSION THREAD

**Discussion by Rtms1984** [openPupil/Open-PupilEXT#3] [Q&A] [answered]
I have a problem while single/stereo camera(s) calibration. The algorithm can't find the pattern. 
![calibration](https://user-images.githubusercontent.com/13430688/128177422-e5f33bfc-fe4f-405e-98b9-da82bad6e8d1.PNG)
![singleCalibration](https://user-images.githubusercontent.com/13430688/128177425-66e29593-e4d1-4910-adaf-105230b7e220.PNG)
As you see, using single camera sharpness, the pattern is found and the average sharpness is more than 3. At the same time, calibration process can't find the pattern. I wonder why is that happening.

**Comment by BZandi** [comment, marked as answer]
Your calibration board seems to be different than the ones that we use, as in your last row there is a black rectangle, which should be a white one. Compare it to our example (see figure). Please go to the calib.io page and try to make a new valid checkerboard (Link: https://calib.io/pages/camera-calibration-pattern-generator).

![Pattern](https://user-images.githubusercontent.com/62992112/128179458-ff433714-15b1-4294-88ed-67ebf2632065.png)
